api/api.yaml

```diff
@@ -0,0 +1,220 @@
+      properties:
+        codec:
+          description: CODEC hex value
+          pattern: 0x([a-f0-9][a-f0-9])+
+          type: string
+        json:
+          description: JSON escaped string
+          type: string
+      title: Arbitrary
+      type: object
+    ArbitraryResponse:
+      description: "An arbitrary instance of a specific type."
+      example:
+        data:
+          codec: '0x66ce79ce829d0c80590356df94f6e958864ed7ac6661b688fe20e6f19f8b4ad48fb9fc0a814d2f71de2e84e7c43fd57bcbd31256d039e00bb08529c940173a03d2eaa7802aa688b35f8a82714b04cd36dff723b781ce61e8cddc98553090a0f068849c1af9e9759948f278f42aa0636b3e5c815d930aec18074205c8c30b221e42260e0f285b91ae7516b00d4c8c2c9f724f912da1def39fe4c45ff3b0caef3c4d18bb2a9f6879224bfa044505e2e3dd55200a4b4e4311505b6fd9953c78652b2ad978c04ef60043af294f81329b4364b5103cff71ec34a7166a18183ad32534ef9ad951d96ce1ec673eedc6f0bb4a55802800a2b9d49db7dd392d5e70e0cd8f5b4f9ee3aab227b61a286ed73a1e7f3f0eb08538657e4f2a683829c83b148405f21cb947443857956afc5b30db8a24d1cf154e9f4ee99e066caaa901571003a97178adf85fed49ef36a99380cd244502'
+          json: '{"bandersnatch": "0x66ce79ce829d0c80590356df94f6e958864ed7ac6661b688fe20e6f19f8b4ad4",
+            "ed25519": "0x8fb9fc0a814d2f71de2e84e7c43fd57bcbd31256d039e00bb08529c940173a03",
+            "bls": "0xd2eaa7802aa688b35f8a82714b04cd36dff723b781ce61e8cddc98553090a0f068849c1af9e9759948f278f42aa0636b3e5c815d930aec18074205c8c30b221e42260e0f285b91ae7516b00d4c8c2c9f724f912da1def39fe4c45ff3b0caef3c4d18bb2a9f6879224bfa044505e2e3dd55200a4b4e4311505b6fd9953c78652b2ad978c04ef60043af294f81329b4364",
+            "metadata": "0xb5103cff71ec34a7166a18183ad32534ef9ad951d96ce1ec673eedc6f0bb4a55802800a2b9d49db7dd392d5e70e0cd8f5b4f9ee3aab227b61a286ed73a1e7f3f0eb08538657e4f2a683829c83b148405f21cb947443857956afc5b30db8a24d1cf154e9f4ee99e066caaa901571003a97178adf85fed49ef36a99380cd244502"}'
+      properties:
+        data:
+          "$ref": "#/components/schemas/Arbitrary"
+      title: ArbitraryResponse
+      type: object
+    CodecParams:
+      description: "Parameters for decoding a CODEC value to JSON."
+      # example:
+      #   codec: "0x66ce79ce829d0c80590356df94f6e958864ed7ac6661b688fe20e6f19f8b4ad48fb9fc0a814d2f71de2e84e7c43fd57bcbd31256d039e00bb08529c940173a03d2eaa7802aa688b35f8a82714b04cd36dff723b781ce61e8cddc98553090a0f068849c1af9e9759948f278f42aa0636b3e5c815d930aec18074205c8c30b221e42260e0f285b91ae7516b00d4c8c2c9f724f912da1def39fe4c45ff3b0caef3c4d18bb2a9f6879224bfa044505e2e3dd55200a4b4e4311505b6fd9953c78652b2ad978c04ef60043af294f81329b4364b5103cff71ec34a7166a18183ad32534ef9ad951d96ce1ec673eedc6f0bb4a55802800a2b9d49db7dd392d5e70e0cd8f5b4f9ee3aab227b61a286ed73a1e7f3f0eb08538657e4f2a683829c83b148405f21cb947443857956afc5b30db8a24d1cf154e9f4ee99e066caaa901571003a97178adf85fed49ef36a99380cd244502"
```

Comment:
What's up with this?

Reply:
to remove. example param is not allowed here in openapi 3

Reply:
removed